apps/portals/src/configurations/genie/resources.ts

```diff
@@ -1,9 +1,6 @@
-export const programSql = 'SELECT * FROM syn36649342'
-export const projectsSql = 'SELECT * FROM syn36649647'
-export const datasetsSql = 'SELECT * FROM syn39041190'
-
 export const contributorsSql = 'SELECT * FROM syn53975138'
 export const genieBPCSql = 'SELECT * FROM syn38806899'
 export const genieDataSql = 'SELECT * FROM syn38806899'
 export const publicationsSql = 'SELECT * FROM syn53974927'
 export const toolsSql = 'select * from syn53975155'
+export const dataReleasesSql = 'select * from syn53977725'
```

Comment:
Add a new Data Releases page, that currently has a table of all data releases (hidden in a new Collapsable wrapper)